src/applications/yellow-ribbon/containers/SearchForm/index.jsx

```diff
@@ -220,13 +220,11 @@ export class SearchForm extends Component {
           </div>
 
           {/* Submit Button */}
-          <button
-            className="usa-button-primary va-button-primary vads-u-width--auto vads-u-padding-y--1p5 vads-u-margin-top--2"
-            disabled={fetching}
-            type="submit"
-          >
-            Search
-          </button>
+          <va-button
+            onClick={onSubmitHandler}
```

Comment:
This web component [has a `submit` prop](https://design.va.gov/storybook/?path=/docs/components-va-button--primary) that I tried to use here (it sets the internal `button` with `type="submit"`) but it did not function as expected. 

I [created an issue](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2379) for the DST to look into it. In the mean time, a workaround is to just use the `onClick` callback to submit the form.